reference.md

```diff
@@ -927,3 +927,23 @@ Field Name | Presence | Type | Description
   }
 }
 ```
+
+
+### deeplink with query params
+
+The uris provided in `realtime_booking` (`android_uri`, `ios_uri`, `web_uri`) and `booking_rules` (`booking_url`) should support a specific set of query params:
+
+Param | Presence | Type | Description
+---|---|---|---
+`pickup_lat` | REQUIRED | Latitude | Latitude of the location where the user will be picked-up.
+`pickup_lon` | REQUIRED | Longitude | Longitude of the location where the user will be picked-up.
+`drop_off_lat` | REQUIRED | Latitude | Latitude of the location where the user will be dropped off.
+`drop_off_lon` | REQUIRED | Longitude | Longitude of the location where the user will be dropped off.
+`pickup_time` | OPTIONAL | Timestamp | Timestamp, in milliseconds, of the moment the user will be picked-up.
+`drop_off_time` | OPTIONAL | Timestamp | Timestamp, in milliseconds, of the moment the user will be dropped-off.
```

Comment:
The Timestamp that we had before this was epoch in seconds. Also we have a Timestamp type with no definition (that was before this PR). Can we fix 🙏

Reply:
Are you refering to the `"last_updated": 1609866247` property?

Reply:
Yes exactly

Reply:
Sorry that I'm a pain, but I was talking adding `Timestamp` inside the `Field Types` section 😬

Reply:
Oh, sorry, I did not get that, pushing the change rn!